src/main/java/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -412,17 +412,6 @@ public IdentifierHolder createParticipant(App app, StudyParticipant participant,
             throwExceptionIfLimitMetOrExceeded(app);
         }
         
-        // Fix for callers that include only externalId and not the new externalId map: in these cases, tools are 
-        // creating the participant before the sign up call (through the Bridge Study Manager). Check and if 
-        // the account with this external ID already exists, return quietly. Otherwise proceed as before.
-        if (participant.getExternalId() != null && participant.getExternalIds().isEmpty()) {
-            AccountId accountId = AccountId.forExternalId(app.getIdentifier(), participant.getExternalId());
-            Account account = accountService.getAccount(accountId); 
-            if (account != null) {
-                return new IdentifierHolder(account.getId());
-            }
-        }
```

Comment:
Sometimes, integration tests need to create participants using External ID, to test the External ID only test cases. Since that bypasses SignUp, should this code remain in ParticipantService?

Reply:
I believe we should switch to using the externalIds map rather than using this externalId field for the integration tests, and it appears we have.